src/main/java/frc/robot/RobotContainer.java

```diff
@@ -167,16 +175,80 @@ public void initializeAutoBuilder() {
   }
 
   public void registerCommands() {
-    NamedCommands.registerCommand("Intake_Note_1.5_Sec", new IntakeNote(intake, index, arm).withTimeout(3));
-    NamedCommands.registerCommand("Outtake_Note_1.5_Sec", new SequentialCommandGroup(
-      new ParallelDeadlineGroup(
-      new InstantCommand(() -> drive.setChassisSpeeds(new ChassisSpeeds(), true)),
+    NamedCommands.registerCommand("Intake_Note_1.75_Sec", new IntakeNote(intake, index, arm).withTimeout(1.75));
```

Comment:
Put any commands that need subsystems in if statements to check if all subsystems needed exist. You can also check the robot id and put all of these commands in one if statement.